notify/notification_apns_test.go

```diff
@@ -2,15 +2,14 @@ package notify
 
 import (
 	"context"
+	"github.com/appleboy/gorush/config"
```

Comment:
Please return these two imports to the block below (just like it was before).

Reply:
done, thanks Ivancho